modules/geo-layers/src/mvt-layer/mvt-layer.js

```diff
@@ -195,6 +249,45 @@ export default class MVTLayer extends TileLayer {
     return deck.pickObjects({x, y, width, height, layerIds, maxObjects});
   }
 
+  _getLayerDataGeomType(data) {
+    if (Promise.resolve(data) !== data) {
+      if (data.points.featureIds.value.length) {
+        return 'points';
+      } else if (data.lines.featureIds.value.length) {
+        return 'lines';
+      } else if (data.polygons.featureIds.value.length) {
+        return 'polygons';
+      }
```

Comment:
I can't remember if the `data` object is guaranteed to have each of these fields, so I wouldn't be surprised if this object accessing would error at times.

Reply:
Can we use `.?` operator in deck.gl yet? Might be a good time to introduce ES2020 - could save us a lot of exceptions here.

Reply:
It cannot be used yet, it throws a webpack exception now.